src/libcore/str/mod.rs

```diff
@@ -143,7 +142,7 @@ impl FromStr for bool {
 
 /// An error returned when parsing a `bool` using [`from_str`] fails
 ///
-/// [`from_str`]: ../../std/primitive.bool.html#method.from_str
+/// [`from_str`]: FromStr::from_str
```

Comment:
Do we support `<bool as FromStr>::from_str`? If not that seems like a good addition, we could also overload `<bool as FromStr>` to link to the trait implementation. No need to do anything for this PR though.

Reply:
We do not